fe/src/main/java/org/apache/doris/analysis/CreateRoutineLoadStmt.java

```diff
@@ -105,6 +105,7 @@
             .add(MAX_BATCH_INTERVAL_SEC_PROPERTY)
             .add(MAX_BATCH_ROWS_PROPERTY)
             .add(MAX_BATCH_SIZE_PROPERTY)
+            .add(LoadStmt.STRICT_MODE)
```

Comment:
should we update our Document about this property?

Reply:
I will update.